test/browsertype-launch-server.spec.ts

```diff
@@ -62,4 +62,17 @@ describe('lauch server', (suite, { wire }) => {
     expect(result['exitCode']).toBe(0);
     expect(result['signal']).toBe(null);
   });
+
+  it('should allow using an existing cdp endpoint', async ({ browserType, server}) => {
```

Comment:
This test don't really prove that the cdp websocket works. @dgozman should we add another bot that runs all the tests via cdp?

Or we can just have a smoke test that.
1. launches a browser server with `--remote-debugging-port`
2. launches a second browser server that connects to the first via cdpWebSocketEndpoint
3. connect to that second browser server and create a page and eval 1+1=2 in there.

Reply:
Yeah, doing that was my first instinct :-) I had the impression the other tests were a bit shallow so I opted to just test the very basics. Happy to change this to a more full-fledged test.

Reply:
Just wondering about the best way to get the `webSocketDebuggerUrl` from `/json/version` (there doesn't seem to be an http client around)

Reply:
You can do `await page.evaluate(() => fetch(url))` and use the browser 😄

Reply:
went with vanilla `http` 😄 

![image](https://user-images.githubusercontent.com/1368633/98167136-e8170200-1ee8-11eb-9e71-67cc150d0c3e.png)

Reply:
@JoelEinbinder added an additional smoke test to ensure the cdp websocket works :-)